### PR TITLE
genai: use SecretStr for api_key

### PR DESCRIPTION
**Pull request by nobu007**
## PR Description

google_api_key of ChatGoogleGenerativeAI is defined as SecretStr.
This PR changes it from str in tests.

## Type

✅ Test

### CODE REVIEW

**Review by lkuligin — APPROVED** [review]

File: libs/genai/tests/integration_tests/test_chat_models.py [hunk 1]
```diff
@@ -1,9 +1,14 @@
 """Test ChatGoogleGenerativeAI chat model."""
+
 import asyncio
 import json
 from typing import Generator, List, Optional, Type
 
 import pytest
+from google.generativeai.types import (  # type: ignore[import]
```
Comment (lkuligin) [inline libs/genai/tests/integration_tests/test_chat_models.py:8]: why do we need this change?

File: libs/genai/tests/integration_tests/test_llms.py [hunk 2]
```diff
@@ -7,9 +7,13 @@
 from typing import Generator
 
 import pytest
+from google.generativeai.types import (  # type: ignore[import]
```
Comment (lkuligin) [inline libs/genai/tests/integration_tests/test_llms.py:10]: why do we need this change?

File: libs/genai/tests/integration_tests/test_chat_models.py (hunk 1 above)
Comment (nobu007) [reply libs/genai/tests/integration_tests/test_chat_models.py:8]: In my environment, this error is shown.
https://github.com/google-gemini/generative-ai-python/issues/264

Do you know or something hint about this?

Comment (lkuligin) [reply libs/genai/tests/integration_tests/test_chat_models.py:8]: please, feel free to add a `# type: ignore[import]` hint but let's continue importing from langchain_google_genai, please

File: libs/genai/tests/integration_tests/test_llms.py (hunk 2 above)
Comment (lkuligin) [reply libs/genai/tests/integration_tests/test_llms.py:10]: same as above

File: libs/genai/tests/integration_tests/test_chat_models.py (hunk 1 above)
Comment (nobu007) [reply libs/genai/tests/integration_tests/test_chat_models.py]: @lkuligin 
from langchain_google_genai._enums import helped me.
It looks import google.ai.generativelanguage_v1beta works without ignore.

**Comment by nobu007** [comment]
I have rebased.

**Comment by nobu007** [comment]
@lkuligin 
Please share error log "langchain-google-genai-us (llm-integration-tests) "?
In my environment, "make integration_test" works fine.
```
35 passed, 9 skipped, 1 xfailed, 13 warnings in 76.23s (0:01:16)
```

**Comment by lkuligin** [comment]
I'm still not sure why we need this PR, linter is green and tests are passing currently.
Could you, please, check that you don't have any extra dependencies, and use exactly the same version as defined in poetry?

**Comment by nobu007** [comment]
Thank you. It worked fine without any change in same version as defined in poetry.
Probably, depends on mypy or somesing version. 
Anyway, I will confirm later about it and llm-integration-tests.

**Comment by nobu007** [comment]
@lkuligin
https://github.com/langchain-ai/langchain-google/pull/465 shows someting problem of llm-integration-tests for me.
Do you have any information?

**Comment by nobu007** [comment]
@lkuligin
PR Description is updated.
It is just test improvement.

**Pull request merged**